testing/test_nose.py

```diff
@@ -366,3 +367,17 @@ def test_method(self):
     assert not reprec.getfailedcollections()
     calls = reprec.getreports("pytest_runtest_logreport")
     assert not calls
+
+
+def test_skip_test_with_unicode(testdir):
+    testdir.makepyfile(
+        """
+        # encoding: utf-8
+        import unittest
+        class TestClass():
+            def test_io(self):
+                raise unittest.SkipTest(u'😊')
```

Comment:
how would this fare against ` unittest.SkipTest(b'\0x01')

Reply:
Works OK I guess:

```
.tmp\test-skip-chinese.py s                                    [100%]
====================== short test summary info ======================
SKIPPED [1] c:\users\bruno\pytest\src\_pytest\nose.py:30:  x01
```

Reply:
probably fails in python2.x with `b'\x21\xef\x1a'` (I picked some haphazard bytes that I assume aren't valid ascii) -- I assume @RonnyPfannschmidt meant to pick some non-ascii bytes? `b'\0x01` is a null byte followed by literal "x01"

Reply:
@asottile yikes, good catch, in my experimentation things seemed fine tho